Implementação do Readme (#7)
julianoboese/Tryitter: Tryitter.Api/README.md

# ✨ Sejam bem vindos ao repositório do projeto Tryitter

Este projeto consiste em criar uma API para ser consumida por uma aplicação que se assemelha a uma rede social próxima ao Twitter.

<details>
<summary><strong>🍀 Endpoints:</strong></summary><br />

  - Fazer o login
  - Buscar todos os posts

>   - Buscar post por id do estudante

Reviewer: ```suggestion
  - Buscar post por id
```

Author: Show

>   - Buscar último post de um estudante

Reviewer: ```suggestion
  - Buscar último post do estudante 🔒 
```

>   - Adicionar post

Reviewer: ```suggestion
  - Adicionar post 🔒 
```

Author: Show

>   - Atualizar post

Reviewer: ```suggestion
  - Atualizar post 🔒 
```

>   - Deletar post

Reviewer: ```suggestion
  - Deletar post 🔒 
```

  - Buscar todos estudantes

>   - Buscar por id um estudante

Reviewer: ```suggestion
  - Buscar um estudante pelo id
```

  - Adicionar um estudante

>   - Atualizar um estudante

Reviewer: ```suggestion
  - Atualizar o estudante 🔒 
```

>   - Deletar um estudante

Reviewer: ```suggestion
  - Deletar o estudante 🔒 
```

Author: show

</details>

### 🔎 Orientações

> 1. Clone o repositório:
> 
>    - `https://github.com/julianoboese/Tryitter`
> 
>    - Entre na pasta do repositório:
> 
>     - `cd Tryitter`

Reviewer: ```suggestion
1. Clone o repositório e entre na pasta do projeto:
    - `cd Tryitter`
```

  

2. Instale as dependências com:
   
     -  `dotnet restore`

3. Suba o container do banco de dados com:
   
     - `docker-compose up -d`

4. Use o comando abaixo para rodar as migrations:
   
   - `dotnet ef database update`
  

>   OBS: este comando irá criar as tabelas POST, STUDENTS e MODULE, também será populada a tabela Module. As outras tabelas do bancos encontram-se vazias, para testar o seu funcionamento, rode aplicação e abra o Swagger para interagir com as rotas.

Reviewer: ```suggestion
  OBS: este comando irá criar as tabelas Posts, Students e Modules e também será populada a tabela Modules. As outras tabelas do bancos encontram-se vazias. Para testar o seu funcionamento, rode aplicação e abra o Swagger para interagir com as rotas.
```

5. Entre na pasta Tryitter.Api:
   
   - `cd Tryitter.Api:`

6. Para rodar a aplicação:
   
   - `dotnet run`

7. O link para o Swagger:

>      - `https://localhost:7143/swagger/index.html`

Reviewer: ```suggestion
   - `https://localhost:7143/swagger/index.html`
```
